app/models/proposal.rb

```diff
@@ -33,6 +33,7 @@ class Proposal < ActiveRecord::Base
   has_many :steps
   has_many :individual_steps, ->{ individual }, class_name: 'Steps::Individual'
   has_many :approvers, through: :individual_steps, source: :user
+  has_many :completers, through: :individual_steps, source: :completer
```

Comment:
I worry that this association name is a but confusing. Yoz was asking me about the concept of completion yesterday, which is why I think of it. It wasn't clear to him exactly what it was.

what about calling this a `step_completer` ? we know that a proposal has many steps, so I think that might be clearer. in the future, we also might want the rename the `approver` relation because a step is not always an approval these days.